students/hyeongu/westagram/settings.py

```diff
@@ -31,6 +31,7 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
 	'corsheaders',
+    'westagram'
```

Comment:
```suggestion
    'django.contrib.staticfiles',
    'corsheaders',
    'westagram'
```
1. 라인 잘 맞춰주세요.
2. 추가하는 app 이름이 westagram이 맞는지 다시 확인해 보세요.

Reply:
1. 라인 정리 완료했습니다!
2. 오기입 수정 완료했습니다. (westagram -> users)